agent/apps/ai_helsabrain/app_code/cores/api_ingress.py

```
@@ -0,0 +1,8 @@
+"""
+هسته API Ingress برای اپلیکیشن
+API Ingress Core for Application
+"""
+
+from app_standards.four_cores import APIIngressCore
```

Ruff (0.12.2)

`app_standards.four_cores.APIIngressCore` imported but unused

Remove unused import: `app_standards.four_cores.APIIngressCore`

(F401)